src/DatabaseLibrary/assertion.py

```diff
@@ -20,7 +20,7 @@ class Assertion(object):
     Assertion handles all the assertions of Database Library.
     """
 
-    def check_if_exists_in_database(self, selectStatement, sansTran=False):
+    def check_if_exists_in_database(self,alias, selectStatement, sansTran=False):
         """
         Check if any row would be returned by given the input `selectStatement`. If there are no results, then this will
```

Comment:
You'd better have to document this change also because this is a breaking change that introduces a new mandatory field.